http-concat.php

```diff
@@ -6,6 +6,9 @@
  * Author: Automattic
  * Version: 1.0
  */
+if ( extension_loaded( 'newrelic' ) ) { // Ensure PHP agent is available
+	newrelic_ignore_transaction();
+}
```

Comment:
This will not work. Requests for actual concat are served from a different endpoint (https://github.com/Automattic/nginx-http-concat/blob/master/ngx-http-concat.php)